source/triggers.txt

```diff
@@ -1,22 +1,105 @@
-========
-Triggers
-========
+.. _triggers:
+.. _triggers-overview:
 
-.. default-domain:: mongodb
+==============
+Atlas Triggers
+==============
 
 .. toctree::
    :titlesonly:
    :caption: Triggers
    :hidden:
 
-   About Triggers </triggers/overview>
    Database Triggers </triggers/database-triggers>
    Authentication Triggers </triggers/authentication-triggers>
    Scheduled Triggers </triggers/scheduled-triggers>
-   Usage Examples </triggers/examples>
+   Resume a Suspended Trigger </triggers/resume-a-suspended-trigger>
+   Disable a Trigger </triggers/disable>
+   Send Trigger Events to AWS EventBridge </triggers/aws-eventbridge>
+   Store New Users in MongoDB </triggers/store-new-users-in-mongodb>
```

Comment:
This is specifically an auth trigger example - maybe we consolidate on that page?

Reply:
actually now that you mention it, this example is very familiar to the example already on that page, except less thorough. i'm deleting the `Store New Users in MongoDB` page.

the Triggers example repo will also have an example of this. don't think we need 3 times.